src/app/pages/task-calendar/rsync/rsync-form/rsync-form.component.ts

```diff
@@ -113,21 +116,22 @@ export class RsyncFormComponent {
         type: 'input',
         name: 'rsync_description',
         placeholder: T('Short Description'),
-        tooltip: T('Optional. Enter an informative description of the\
-                    new rsync task.'),
+        tooltip: T('Enter an informative description of the new rsync\
+                    task.'),
       }, {
         type: 'scheduler',
         name: 'rsync_picker',
         placeholder: T('Schedule the Rsync Task'),
-        tooltip: T('Choose one of the convenient presets\
-          or choose <b>Custom</b> to trigger the advanced scheduler UI'),
-          required: true
+        tooltip: T('Choose one of the convenient presets or\
+                    <i>Custom</i> to open the advanced cron job\
+                    scheduler UI.'),
+        required: true
       }, {
         type: 'checkbox',
         name: 'rsync_recursive',
         placeholder: T('Recursive'),
         tooltip: T('Set to include all subdirectories of the specified\
-                    pool during the rsync task.'),
+                    location during the rsync task.'),
```

Comment:
s/location/directory/.  Also, "during the rsync task" is not needed.  So:

Include all subdirectories of the specified directory. When unset, only the specified directory is included.

Reply:
done